src/main/java/io/github/thebusybiscuit/slimefun4/implementation/tasks/RadioactivityTask.java

```diff
@@ -0,0 +1,142 @@
+package io.github.thebusybiscuit.slimefun4.implementation.tasks;
+
+import io.github.thebusybiscuit.slimefun4.api.player.PlayerProfile;
+import io.github.thebusybiscuit.slimefun4.core.attributes.ProtectionType;
+import io.github.thebusybiscuit.slimefun4.core.attributes.Radioactive;
+import io.github.thebusybiscuit.slimefun4.implementation.SlimefunPlugin;
+import io.github.thebusybiscuit.slimefun4.implementation.items.RadioactiveItem;
+import io.github.thebusybiscuit.slimefun4.utils.itemstack.ItemStackWrapper;
+import me.mrCookieSlime.Slimefun.Objects.SlimefunItem.SlimefunItem;
+import me.mrCookieSlime.Slimefun.api.SlimefunItemStack;
+import net.md_5.bungee.api.ChatMessageType;
+import net.md_5.bungee.api.chat.ComponentBuilder;
+import org.bukkit.Bukkit;
+import org.bukkit.ChatColor;
+import org.bukkit.GameMode;
+import org.bukkit.Material;
+import org.bukkit.entity.Player;
+import org.bukkit.inventory.ItemStack;
+import org.bukkit.potion.PotionEffect;
+import org.bukkit.potion.PotionEffectType;
+
+import javax.annotation.Nonnull;
+import java.util.HashMap;
+import java.util.Set;
+import java.util.UUID;
+
+/**
+ * The {@link RadioactivityTask} handles radioactivity for
+ * {@link Radioactive} items.
+ *
+ * @author Semisol
+ *
+ */
+public class RadioactivityTask implements Runnable {
+    public static HashMap<UUID, Double> radioactivityLevel = new HashMap<>();
+    private final PotionEffect WITHER = new PotionEffect(PotionEffectType.WITHER, SlimefunPlugin.getCfg().getOrSetDefault("options.radiation-update-interval", 1) * 20 + 20, 1);
+    private final PotionEffect WITHER2 = new PotionEffect(PotionEffectType.WITHER, SlimefunPlugin.getCfg().getOrSetDefault("options.radiation-update-interval", 1) * 20 + 20, 4);
+    private final PotionEffect BLINDNESS = new PotionEffect(PotionEffectType.BLINDNESS, SlimefunPlugin.getCfg().getOrSetDefault("options.radiation-update-interval", 1) * 20 + 40, 0);
+    private final PotionEffect SLOW = new PotionEffect(PotionEffectType.SLOW, SlimefunPlugin.getCfg().getOrSetDefault("options.radiation-update-interval", 1) * 20 + 20, 3);
+    @Override
+    public void run() {
+        for (Player p: Bukkit.getOnlinePlayers()){
```

Comment:
Space after the colon
```suggestion
        for (Player p : Bukkit.getOnlinePlayers()){
```